Bottom: 8754908f54691ebf8eb338a5cf23c230a90d34d4
Top:    6a80dbc9a9422453e6bcd2724cd614a7768cb5e7
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-25 12:02:40 -0700

Refresh of dbg

---

diff --git a/src/preload.c b/src/preload.c
index 6f8e4f8..bc95da8 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -438,7 +438,7 @@ int accept(int socket, struct sockaddr *addr, socklen_t *addrlen)
 		ret = (type == fd_rsocket) ? raccept(fd, addr, addrlen) :
 					     real.accept(fd, addr, addrlen);
 		printf("accept %d, new index %d new socket %d err %s\n",
-			socket, index, ret, strerr(errno));
+			socket, index, ret, strerror(errno));
 		if (ret < 0) {
 			fd_close(index, &fd);
 			return ret;
